tests/fixture-data/net-runner/deploying-a-runtime-staking.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Do we really need this? (#3110)

Reply:
Uhm, yeah i can instead extend the the `default fixture` command with a "fund entities" flag, and this can be done automatically, that will make it much more managable.

Reply:
i'll update once https://github.com/oasisprotocol/oasis-core/pull/3135 lands